app/src/main/java/com/courseproject/tindar/ui/chat/ChatActivity.java

```diff
@@ -170,7 +171,12 @@ public void sentMessage(View v){
         MessageModel newMessage = new TindarMessage(input, new Timestamp(System.currentTimeMillis()),
                 this.userId, this.otherUserId);
         this.loadedMessages.add(newMessage);
+        this.chatInput.getText().clear();
 
         this.adapter.notifyDataSetChanged();
     }
+
+    public void backButtonPressed(View v){
```

Comment:
This is not override?

Reply:
No, it's not overriding anything at the moment. I could in theory create an interface for the xml file, but it doesn't seem worth it right now.